integration-tests/opentelemetry-logging/src/test/resources/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+
+services:
+  otel-collector:
+    image: otel/opentelemetry-collector-contrib
```

Comment:
there should be additional assertions for the tests under the IT project [opentelemetry-vertx-exporter](https://github.com/quarkusio/quarkus/blob/b6f54993c43f955f2b20855b9c4d694af7c210f2/integration-tests/opentelemetry-vertx-exporter/src/test/java/io/quarkus/it/opentelemetry/vertx/exporter)

Reply:
Done, I added test in the opentelemetry-vertx-exporter